tests/fdb/test_fdb.py

```diff
@@ -165,6 +165,9 @@ def test_fdb(ansible_adhoc, testbed, ptfadapter, duthost, ptfhost, pkt_type):
     1. verify fdb forwarding.
     2. verify show mac command on DUT for learned mac.
     """
+    
+    if testbed['topo']['type'] in ['t1', 'ptf']:
```

Comment:
Can we do a white-list like we did in the ansible version of this test? It helps ensure that if we add a new topo we don't accidentally run this test on an unsupported topo. :)

https://github.com/Azure/sonic-mgmt/blob/50a6bc19ac435f8a270d707238d2e1f2a8cc084e/ansible/roles/test/vars/testcases.yml#L178